_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,205 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.4.9: # Images of Text (No Exception) (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+    - Times Square image released into the public domain by (WT-shared) Ypsilonatshared at wts wikivoyage.
+---
+
+## Applicability
+
+This rule applies to any [visible][] [embedded image][] that is not [marked as decorative][].
+
+## Expectation
+
+For each of the image resources referenced by the [image sources][] of the test target, any of the following is true:
+
+- the image resource does not contain text expressing anything in a [human language][]; or
+- the text in the image resource is not the most significant visual content; or
+- it is [essential][] that the text in the image resource is rendered with that specific presentation.
+
+## Assumptions
+
+_There are currently no assumptions_
```

Comment:
I'm starting to think about a nasty corner case where 2 images are overlapping, the "bottom" one contains text, but the "top" one covers all the text, leaving a bit of non-text visible (e.g. a bit of the "background" of the bottom image).

Then, the bottom image would be applicable (it is visible because bit of its background is) and fail the rule (because of its text) but wouldn't fail the SC because the text is actually not visible…

=> Should there be an assumption that text in the image is visible? I don't think we can use our definition of "visible" since making part of a .jpg transparent is not really a thing. That should be corner case enough that it doesn't really happen often, so an assumption should be enough.



=> I'm also starting to freak out a bit that a similar situation happens with other rules that apply to "visible" stuff.

Reply:
Good catch. I'm adding the assumption, and later we can think about adding a similar assumption to other rules in need.